docs/UserGuide.md

```diff
@@ -20,16 +20,19 @@ It is optimized for a _Command Line Interface_ (CLI), but you can still use the
 
 ## Getting Started!
 
-> [!IMPORTANT]
+> [IMPORTANT]
 > Ensure you have Java `11`  installed in your computer. If you do not have it installed, download it from [here.](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html)
 1. Download the latest `internbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
 
 2. Copy the file to the folder you want to use as the _home folder_ for your Internbook.
 
 3. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar internbook.jar` command to run the application.
-> [!TIP]
+> [TIP]
```

Comment:
same goes to the [TIP], just a suggestion maybe it could be **TIP:** for better readability